x-pack/plugins/ml/public/file_datavisualizer/components/import_view/importer/csv_importer.js

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+
+import { Importer } from './importer';
+import { parse } from './csv_parse';
+// import csvParse from 'csv-parse';
```

Comment:
Can this line go?

Reply:
it's still part of the other commented out code below. I'm reluctant to remove it until i'm satisfied  can't get the parsing to run faster